docs/CONTRIBUTING.md

```diff
@@ -1,11 +1,20 @@
-# Contribution guidelines [WIP]
+# Contribution guidelines 
 
-## How to contribute to project? [Coming soon]
+## How to contribute to project? 
+All external contribution to this project is welcome. To propose changes, we encourage contributors to open a new branch and propose their changes by opening a new PR. 
 
 ## Having problems?
-Have you encountered an error? A critical step in troubleshooting is being able to reproduct the problem. Instructions to publicly reproduce errors using GitHub Actions can be found in our [guide to reproduce errors.](/docs/REPRODUCE_ERRORS.md)
+Have you encountered an error? A critical step in troubleshooting is being able to reproduce the problem. Instructions to publicly reproduce errors using GitHub Actions can be found in our [guide to reproduce errors.](/docs/REPRODUCE_ERRORS.md)
 
-## Code template [Coming soon]
+## Code template
+
+###Issue and PR templates
```

Comment:
```suggestion
### Issue and PR templates
```

Reply:
I would remove "issue and PR templates" from the code template section, as that doesn't really fit, and instead just move the text under the "How to contribute to the project?" section right after the mention of the pull request. And given that GitHub forces you to do most of this, I think this text can be reduced to:

>We encourage contributors to format pull request titles following the [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)